src/shipping/strategies/amazon/amazon-pay-shipping-strategy.spec.ts

```diff
@@ -269,4 +269,22 @@ describe('AmazonPayShippingStrategy', () => {
         expect(store.dispatch).toHaveBeenCalledWith(action);
         expect(output).toEqual(store.getState());
     });
+
+    it('updates consignment shipping address with custom fields for amazon pay on strategy address update', async () => {
```

Comment:
you are already on `AmazonPayShippingStrategy.spec` file, this could simply say
`updates address with provided custom fields and existing address`